doc/api/stream.md

```diff
@@ -1338,6 +1340,18 @@ run().catch(console.error);
 rs.resume(); // drain the stream
 ```
 
+Note that `stream.finished` leaves dangling event listeners (in particular
+`'error'`, `'end'`, `'finish'` and `'close'`) after `callback` has been
+invoked. If this unwanted behaviour then the returned cleanup function
```

Comment:
It’s important to note the reason for this, ‘error’ can be emitted multiple times, and we don’t want to crash.

Reply:
Fixed